RELEASE.md

```diff
@@ -50,7 +50,7 @@ yarn install
 * And last (but not least 😁) do your release.
 
 ```sh
-npx release-it
+pnpm release-it
```

Comment:
The PNPM equivalent to execute a binary of a package installed locally is `pnpm exec`.

```suggestion
pnpm exec release-it
```

Reply:
I believe either of these will work, according to their docs for the `exec` command, it isn't required if there is no conflict with a built-in pnpm command.

https://pnpm.io/cli/exec
Since either way works, I think it is more of a preference. I can do whichever matches the project's styles better.

Reply:
Thanks a lot for clarifying. I guess I mixed it up with NPM.

Ember CLI blueprints are using the shorthand without `run`: https://github.com/ember-cli/ember-cli/blob/d3b19b5f3db1c811273306f493e1ea3da0c70c74/blueprints/addon/index.js#L180 I think we should follow that convention.